### ISSUE TITLE
Implement Warmup

### ISSUE THREAD

**Issue by gwaybio** [greenelab/tybalt#12]
As discussed in [Sonderby et al. 2016](https://arxiv.org/abs/1602.02282) a warm up period from deterministic to stochastic autoencoder model as training epochs proceed may enhance parameter usage. 

This [stackoverflow post](https://stackoverflow.com/questions/42787181/variationnal-auto-encoder-implementing-warm-up-in-keras) may be useful to review.
